### PR TITLE
Adds in Extraction Cargo Console

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds in the Cargo console.
Removes almost medipens on maps, and adds the cargo console into extraction.
Removes half the clerk kit. it can now be found in the cargo console.

UI done by @InsightfulParasite.

## Why It's Good For The Game
Cleans out Clerk gear, and it's new content for the EO to mess about with.

## Changelog
:cl:
add: Added Extraction cargo console
del: Removed half of Clerk spawning gear
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/lightreplacer.dm [hunk 1]
```diff
@@ -54,8 +54,8 @@
 	slot_flags = ITEM_SLOT_BELT
 	force = 8
 
-	var/max_uses = 20
-	var/uses = 10
+	var/max_uses = 200
+	var/uses = 100
```
Comment (dsmith328) [inline code/game/objects/items/devices/lightreplacer.dm:58]: Best change 💯

**Review by dsmith328 — COMMENTED** [review]
From what I can see, most things here work for what we need. Prices and items in these lists can be changed later. A prime example is if Bolas become an issue, they can be removed.

**Review by dsmith328 — APPROVED** [review]

**Comment by Kitsunemitsu** [comment]
Waiting on checks from rebase and then merging

**Comment by dsmith328** [comment]
If it's of real concern, we can wait for Egor to verify the list's contents

**Pull request merged**